cli.js

```diff
@@ -29,32 +34,66 @@ var usage = `Usage
 Work in progress! Learn more at github.com/cabal-club
 `
 
-if (args.key) {
-  args.key = args.key.replace('cabal://', '').replace('cbl://', '').replace('dat://', '').replace(/\//g, '')
-  args.db = rootdir + args.key
+// Attempt to load local or homedir config file
+try {
+  var config
+  var configFilename = '.cabal.yml'
```

Comment:
could we support this also in the cabal core api?

Reply:
I think that's a great idea! Or maybe as a part of the `cabal-client` library?

Reply:
+1 for cabal-client. I'd kinda like cabal-core to stay a config-less database, and move user preferences into a thin cabal-client on top